.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@
 Do not describe the purpose here but:
 
 - [ ] Short descriptions should be in the release notes (added as entry in
-      doc/news/_preparation_next_release.md which contains `*(my name)*`)
+      `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
```

Comment:
Any specific reason to go from * to _?

Reply:
Not really, `_` seems to be the default character Prettier uses for italics. As far as I can tell, we used both `_` and `*` until now. 

I do like the more explicit difference in the updated style between 

- _italics_ (`_italics_`),
- **bold style** (`**bold**`), and
- list items (`- item` and **not** `* item`) 

though.